.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+[Back](./README.md)
+
+# Contributing
+
+## A good abbreviation is:
+
+-  `global` • understood by everyone
+-  `significant` • seen for the first time you still know what it means
+
```

Comment:
I think we should add more info here, but it's not blocking now.

Reply:
Yeah, but for now I don't have other ideas. Maybe the "how to abbr" with letters or syllables.